### PR TITLE
fix bug in two_stage_pp, reduce training memory

### PR DESCRIPTION
**Pull request by yuhuan-wu**
Hello Tianwei,

Since BatchNorm1d exists on the reader of the PointPillars. In the two-stage training of CenterPoint-Pillars, BatchNorm1d should also be converted to FrozenBatchNorm, otherwise it will keep updating.

Moreover, I find a minor issue that causes inconsistent memory usage in distributed training of the two-stage. GPU 0 will occupy 600MB memory more.  It seems that the code will load the pretrained model on GPU as default. So loading it on CPU as default will save more memory on GPU 0.

### CODE REVIEW

**Comment by tianweiy** [comment]
Looking great. I am travelling in the mountains and will merge once I get back

**Comment by tianweiy** [comment]
LGTM, thanks @yuhuan-wu !
